### ISSUE TITLE
Something went wrong while saving this configuration: Area is already set

### ISSUE THREAD

**Issue by Swapnil0411** [magento/magento2#18266]
### Preconditions

1. When you change the theme from Magento admin panel . Content >> Configuration  >> Edit >>  Applied Theme. After saving then it shows exception : "Something went wrong while saving this configuration: Area is already set". (Magento 2.2.4 and Magento 2.2.5 version only.)

### Steps to reproduce

1.login to Magento admin panel.
2.Navigate to the content >> Configuration >> Edit >> Applied theme.
3. After click on save  configuration button it showing error:  "Something went wrong while saving this configuration: Area is already set". 
### Solution 
1. Need to override or create a plugin.
2.Change in Magento\Email\Model\AbstractTemplate.php this: Line Number 535
public function setForcedArea($templateId)
{
    if ($this->area) {
        throw new \LogicException(__('Area is already set'));
    }
    $this->area = $this->emailConfig->getTemplateArea($templateId);
    return $this;
}
To 
public function setForcedArea($templateId)
{
    if (!isset($this->area)) {
        $this->area = $this->emailConfig->getTemplateArea($templateId);
    }
    return $this;
}

### Expected result

1. [Screenshots, logs or description]
[2.]
![default store view design content magento admin](https://user-images.githubusercontent.com/38105172/46085324-601b8580-c1c3-11e8-8061-3829abfab255.png)

### Actual result

1. [Screenshots, logs or description]
2.
![default store view design content magento admin 1](https://user-images.githubusercontent.com/38105172/46085418-a07b0380-c1c3-11e8-963a-8774ebdf8b18.png)

**Comment by magento-engcom-team** [comment]
Hi @Swapnil0411. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@Swapnil0411 do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by ghost** [comment]
Hi @Swapnil0411  thank you for you report, This is a duplicate of https://github.com/magento/magento2/issues/14968
Closing this one. Please read that other thread, it has some pointers to patches which will resolve this issue. 
Issue will also be fixed in Magento 2.2.6
